package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "lint": "eslint . --ext .js,.jsx",
     "prebuild": "npm run lint",
-    "build": "gulp && npm run spellcheck && ./gradlew :backpack-compose:generateEverything",
+    "build": "gulp && npm run spellcheck && ./gradlew generateTokens",
```

Comment:
this runs the task from all modules that have it

Reply:
There might be a module in the future which will not have the task. Can we explicitly declare the task at the top level?

Reply:
Not all modules need to have it declared - it just detects the ones that have it and runs it. I didn't have it defined in common initially until I remembered we need it there